src/apps/settings/src/app/views/Metrics/Metrics.js

```diff
@@ -93,22 +95,28 @@ export default function Metrics(props) {
   return (
     <>
       <section className={styles.MetricsHeader}>
-        <ButtonGroup className={styles.BtnGroup}>
+        <ButtonGroup variant="contained">
```

Comment:
Same concern regarding ButtonGroup

Reply:
This is another view with related button groups and decided to utilized MUI's ButtonGroup 

Legacy
![Screen Shot 2022-05-09 at 12 56 53 PM](https://user-images.githubusercontent.com/22800749/167489061-48d7bc88-028f-4f25-a05c-8f6e7bd0f0f2.png)

Mui's Button Group screenshot 
![Screen Shot 2022-05-09 at 12 57 20 PM](https://user-images.githubusercontent.com/22800749/167489099-7857dd13-4432-4f6c-90f9-be5127bdc2db.png)

CC: @shrunyan  feedback on this one as well please. Keep stand alone buttons or is the ButtonGroup okay?

Reply:
Same. Lets go with the change, using the MUI button group